themes/default/content/docs/guides/saml/onelogin.md

```diff
@@ -93,7 +93,8 @@ First, navigate to the OneLogin Application you created above and click the **Mo
 
 1. Open the file and copy the entire block of XML text in your clipboard
 1. Open the Pulumi Service and navigate to your SAML organization.
-1. Select the **Settings** tab, and then select **SAML SSO**.
+1. Select the **Access management** view, and then select **Change requirements**.
```

Comment:
i think we should still say settings here, because they have to go to org settings... maybe...

```suggestion
1. Select **Access management** from organization settings, and then select **Change requirements**.
```

Reply:
I updated the onelogin saml and scim pages to match the verbage used on okta page.